#! /bin/sh /usr/share/dpatch/dpatch-run
## 06-revert-mpc-kokyu-change.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: revert changeset 79309 that broke Kokyu library search path

@DPATCH@
--- ACE_wrappers.orig/bin/MakeProjectCreator/modules/AutomakeWorkspaceHelper.pm
+++ ACE_wrappers/bin/MakeProjectCreator/modules/AutomakeWorkspaceHelper.pm
@@ -114,8 +114,8 @@
   my($reldir)  = shift;
   my($libname) = shift;
 
-  if ($libname =~ /^lib(ace)/i) {
-    if (!defined $reldir || $reldir eq '') {
+  if ($libname =~ /^lib(ace|kokyu\.)/i) {
+    if (!defined $reldir) {
       if (defined $libdirs{$libname}) {
         $reldir = $libdirs{$libname};
       }
@@ -126,18 +126,6 @@
     $str =~ s!$libname!\$(ACE_BUILDDIR)/$reldir/$libname!;
     return $str;
   }
-  elsif ($libname =~ /^lib(kokyu\.)/i) {
-    if (!defined $reldir || $reldir eq '') {
-      if (defined $libdirs{$libname}) {
-        $reldir = $libdirs{$libname};
-      }
-      else {
-        $reldir = 'Kokyu';
-      }
-    }
-    $str =~ s!$libname!\$(top_builddir)/$reldir/$libname!;
-    return $str;
-  }
   elsif ($libname =~ /^lib(tao|kokyu)/i) {
     if (defined $reldir) {
       $reldir =~ s!TAO/!!;
